proto/expansion.proto

```diff
@@ -18,9 +18,10 @@ message Expansion {
   repeated uint32 costs = 1 [packed=true];
   repeated uint32 durations = 2 [packed=true];
   repeated uint32 distances = 3 [packed=true];
-  repeated EdgeStatus edge_status = 4;
+  repeated EdgeStatus edge_status = 4 [packed=true];
```

Comment:
this is technically a breaking change but its a beta service so oh well i guess

Reply:
maybe it's time after this PR to take it out of BETA state?